citrixitm/resource_citrixitm_dns_app_test.go

```diff
@@ -272,3 +277,57 @@ func testAccCheckCitrixITMDnsAppDestroy(s *terraform.State) error {
 
 	return nil
 }
+
+func TestDisabledAppProducesErrorOnRead(t *testing.T) {
+	appID := 123
+	mux := http.NewServeMux()
```

Comment:
what's `mux` -- is this terraform specific terminology?

Reply:
Mux is short for "multiplexer". It's not Terraform specific. It's found in most example Go code abbreviated that way.